src/repository/Repository.ts

```diff
@@ -339,14 +339,14 @@ export class Repository<Entity extends ObjectLiteral> {
     /**
      * Increments some column by provided value of the entities matched given conditions.
      */
-    increment(conditions: FindConditions<Entity>, propertyPath: string, value: number | string): Promise<UpdateResult> {
+    increment(conditions: FindConditions<Entity>, propertyPath: Extract<keyof Entity, string> | string, value: number | string = 1): Promise<UpdateResult> {
```

Comment:
its opinionated if we should make `1` default or not. Need some additional opinions on this small change.